tests/e2e/test_get_transfers.py

```diff
@@ -1,7 +1,7 @@
 import unittest
```

Comment:
General discussion topic not specific to this line:

In my humble opinion, I feel like we have way to many e2e tests as part of our test suite and that most often a lot of these tests seem like integration tests than actual e2e tests. E.g.: `test_get_block_number.py` `test_get_transfers.py` `test_post_transaction.py` `test_prices.py` `test_slack_post.py` `test_token_details.py` feel like integration tests to me. What's your take on that?

Even then, this opens a place for a new discussion that some of these tests could be turned into unit tests by starting to mock external dependencies which are often used in the tests above. 

In my opinion would think of creating one e2e test which would mimick the actual tuesday call for our solvers and then have the rest turned into functional tests and/or mocked unit tests where only our part of the code is tested and we assume that the API is something we expect to be working.

Refs: https://www.geeksforgeeks.org/types-software-testing/ https://www.atlassian.com/continuous-delivery/software-testing/types-of-software-testing

Reply:
Yes this sounds reasonable. We just need to find some time to do this stuff.

Reply:
> Yes this sounds reasonable. We just need to find some time to do this stuff.

This is the only thing I feel like we've been lacking a lot recently :sweat_smile:

Reply:
Issue here: https://github.com/cowprotocol/solver-rewards/issues/105

Reply:
> Issue here: #105

Thanks!